openlibrary/plugins/openlibrary/js/ile/utils/SelectionManager/SelectionManager.js

```diff
@@ -138,7 +141,11 @@ export default class SelectionManager {
             if (action.requires_type.every(type => this.selectedItems[type].length > 0)) {
                 action.applies_to_type.forEach(type => items.push(...this.selectedItems[type]));
                 if (action.multiple_only ? items.length > 1 : items.length > 0)
-                    this.ile.$actions.append($(`<a target="_blank" href="${action.href(this.getOlidsFromSelectionList(items))}">${action.name}</a>`));
+                    if (action.href) {
+                        this.ile.$actions.append($(`<a target="_blank" href="${action.href(this.getOlidsFromSelectionList(items))}">${action.name}</a>`));
+                    } else if (action.onclick && action.name === 'Tag Works') {
+                        this.ile.$actions.append($(`<a id="tag_multiple_works">${action.name}</a>`).on('click', () => action.onclick(this.getOlidsFromSelectionList(items))));
```

Comment:
Something like this might be used to render this if in a vue component:

```suggestion
                        this.ile.$actions.append($(`<a id="tag_multiple_works">${action.name}</a>`)
                        .on('click', () => this.renderBulkTagDropper());
```

```js
async function renderBulkTagDropper() {
    // If not loading the JS from the python-side
    // await import('build/components/production/ol-bulk-tag-dropper.min.js');

    document.body.appendChild(document.createElement('ol-bulk-tag-dropper'));
}
```